lib/Rex/Commands/Sysctl.pm

```diff
@@ -79,6 +94,11 @@ sub sysctl {
       Rex::Logger::debug("$key has already value $val");
     }
 
+    if ( exists $options{persistent} && $options{persistent} eq "true" ) {
```

Comment:
I was thinking about this but I think I should add a way to remove the sysctl from /etc/sysctl.conf.

That could be changed to 

```
if ( exists $options{persistent} )
  if ( $options{persistent} )
    sysctl_save;
  else
    sysctl_remove;
}
```
Better ?